passenger.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class passenger{
+    private String name;
+    private int number;
+    String membership;
+    private List<activity> activities;
+    float balance;
+    
+    public passenger(String name, int number, String membership, float balance){
+        this.name = name;
+        this.number = number;
+        this.membership = membership;
+        this.activities = new ArrayList<activity>();  
```

Comment:
Activities is a type you already created above. Which is a better name? Signed activities or just activities, why?